File: primes.vb
Module primes
    Sub Main()
        For i = 2 To 100
            If isPrime(i) Then
                Console.WriteLine(i)
            End If
        Next
    End Sub
    Function isPrime(ByVal x As Integer) As Boolean
        Dim y As Integer
        y = x / 2
        For i = 2 To y
            If x Mod i = 0 Then
                Return False
            End If
        Next
        Return True
    End Function
End Module

File: primes.pas
var
    a: boolean;
    i: integer;
    j: integer;
begin
    for i := 1 to 100 do
    begin
        a := true;
        for j := 2 to (i - 1) do
        begin
            if((i mod j) = 0) then
                a := false;
        end;
        if(a) then
            writeln(i);
    end;
end.